Plugins/SequencePlugin/Source/SequencePlugin/Private/Types/ERC721.cpp

```diff
@@ -0,0 +1,104 @@
+#include "Types/ERC721.h"
+#include "ABI/ABI.h"
+
+FRawTransaction UERC721::MakeGrantRoleTransaction(const FString& role, const FString& ToAddress)
+{
+	FString FunctionSignature = "grantRole(bytes32,address)";
+
+	FString Role = role;
+
+	TArray<uint8> ProofBytes;
+	ProofBytes.SetNumUninitialized(32);
+	FMemory::Memcpy(ProofBytes.GetData(), TCHAR_TO_UTF8(*Role), FMath::Min(Role.Len(), 32));
```

Comment:
When granting a role to a smart contract, you typically grant the role to `keccak-256(role_name)`. Is this what this code is doing? If so, we should include it as a keccak256 hash function helper somewhere in the SDK

Reply:
when youre missing the role to do smth. the response actually gives you the role converted, so the fstring is to input the role converted already. we can do a utility function to make the conversion,